pkg/operator/ceph/controller/spec.go

```diff
@@ -601,6 +603,40 @@ fi`,
 	}
 }
 
+// GenerateLivenessProbeExecDaemon generates a liveness probe that makes sure a daemon has a socket,
```

Comment:
Typo:  this is a readiness probe , not a liveness one

Reply:
Fixed